apps/src/templates/instructions/TopInstructionsHeader.jsx

```diff
@@ -64,17 +64,16 @@ function TopInstructionsHeader(props) {
       isMinecraft={isMinecraft}
     >
       <div style={styles.paneHeaderOverride}>
-        {/* For CSF contained levels we use the same audio button location as CSD/CSP*/}
```

Comment:
These changes just move around the collapse/expand icon and the inline audio icon to match their reading order on screen. They're absolutely positioned, so this does not affect their presentation.

Reply:
Great change :)

Reply:
do things still work as expected in right to left?

Reply:
Yep! as far as I can tell